### PR TITLE
Improve `Feature` mapping in automatically generated changelog

### PR DESCRIPTION
**Pull request by getdave**
## Description
In https://github.com/WordPress/gutenberg/pull/33229/ we introduced a mechanism which attempts to automatically assign PRs to a given "Feature" within each section.

Previously this task was completed manually so this was a good step forward.

Note we do not need to manually map all features. Anything marked with a Block specific label will be mapped to `Block Library`. Similarly anything labelled with `[Feature] X` label will be automatically mapped to a feature named `X` (or whatever the name of the feature in the label is).

Despite this however, the mapping is currently rather bare bones and needs augmenting. Some `[Feature]` labels are best mapped to a specific feature. Manual intervention is required.

To this end this PR adds additional mapping of Github labels to the feature we feel best matches.

Please feel free to submit additional mappings either by:

* Pushing to this PR.
* Using Github's "Suggestions" feature to submit changes.

The mapping is easily improved by cross referencing the human-made mapping that @gwwar undertook for `11.1.0`. Let's look at how she mapped the PRs to features and cross reference the labels. From that we shoudl be able to tell which labels are best mapped to which features.

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] I've tested my changes with keyboard and screen readers. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR (please manually search all `*.native.js` files for terms that need renaming or removal).

### CODE REVIEW

File: bin/plugin/commands/changelog.js [hunk 1]
```diff
@@ -127,6 +133,21 @@ const LABEL_FEATURE_MAPPING = {
...
+	'[a11y] Zooming': 'Accessibility',
+	'[Package] E2E Tests': 'Testing',
+	'[Package] E2E Test Utils': 'Testing',
+	'Automated Testing': 'Testing',
+	'CSS Styling': 'Style and Layout',
+	'developer-docs': 'Documentation',
+	'[Type] Documentation': 'Documentation',
+	'Global Styles': 'Global Styles',
+	'[Type] Build Tooling': 'Build Tooling',
+	'npm Packages': 'Plugin',
```
Comment (priethor) [inline bin/plugin/commands/changelog.js]: ~~NPM~~`npm` packages have been grouped in different ways, including Workflow, Plugin, and ~~NPM~~`npm` packages. I would advocate for the latter as they are versioned and published independently to the plugin but would love to hear @gziolo's thoughts on this.

File: bin/plugin/commands/changelog.js [hunk 2]
```diff
@@ -127,6 +133,21 @@ const LABEL_FEATURE_MAPPING = {
...
 	'New Block': 'Block Library',
+	'Accessibility (a11y)': 'Accessibility',
+	'[a11y] Color Contrast': 'Accessibility',
+	'[a11y] Keyboard & Focus': 'Accessibility',
+	'[a11y] Labelling': 'Accessibility',
+	'[a11y] Zooming': 'Accessibility',
+	'[Package] E2E Tests': 'Testing',
+	'[Package] E2E Test Utils': 'Testing',
+	'Automated Testing': 'Testing',
+	'CSS Styling': 'Style and Layout',
```
Comment (priethor) [inline bin/plugin/commands/changelog.js]: Do we have an example of "Style and Layout"? I haven't seen "Style and Layout" as of recently, and most recent PRs that would have fallen in this category were not related to Layout.

**Review by priethor — COMMENTED** [review]
Love to see more improvements in this direction!

Not directly tied to this PR, but one more thing we need to reevaluate is the precedence of feature types. For example, because the `sortType` function gives more precedence to labels with a lower index in `LABEL_TYPE_MAPPING`, a PR with both `[Type] Bug` and `Documentation` will get grouped with bugs. In practice, this has been the other way around in recent releases.

File: bin/plugin/commands/changelog.js (hunk 2 above)
Comment (getdave) [reply bin/plugin/commands/changelog.js]: Unlike most of the other choices that were carefully gleaned from assessing against the 11.1.0 changelog, I think I just went ahead made this one up 😄 

Open to alternative suggestions...

Comment (priethor) [reply bin/plugin/commands/changelog.js]: `CSS & Styling` has been used a few times, I would suggest we go with that one until we come up with something better 👍

Comment (getdave) [inline bin/plugin/commands/changelog.js]:
```suggestion
	'CSS Styling': 'CSS & Styling',
```

File: bin/plugin/commands/changelog.js (hunk 1 above)
Comment (getdave) [inline bin/plugin/commands/changelog.js]:
```suggestion
	'npm Packages': 'NPM Packages',
```

**Comment by getdave** [comment]
@gziolo @priethor This now just needs a ✅ in order to merge.

File: bin/plugin/commands/changelog.js [hunk 3]
```diff
@@ -146,7 +146,7 @@ const LABEL_FEATURE_MAPPING = {
 	'[Type] Documentation': 'Documentation',
 	'Global Styles': 'Global Styles',
 	'[Type] Build Tooling': 'Build Tooling',
-	'npm Packages': 'Plugin',
+	'npm Packages': 'NPM Packages',
```
Comment (gziolo) [inline bin/plugin/commands/changelog.js]: Nit: it's npm not NPM :)

Screen Shot 2021-07-23 at 10 11 28

**Review by gziolo — APPROVED** [review]

**Pull request merged**